installation/index.md

```diff
@@ -41,10 +41,11 @@ You could also [download Azul Zulu](https://www.azul.com/downloads/zulu-communit
 Oracle Java is also suitable for most configurations but it's not recommended. Licensing restrictions may apply.
 
 ::: warning
-Java 17 is recommended for openHAB.
-Using Java 21 is possible, but still considered experimental.
+Java 17 is recommended for openHAB 4.x series.
 OpenHABian can install Java 21, but only on the native 64-bit image.
 Please note that versions of Java other than 17 and 21 are not supported at the moment.
+
+Upcoming openHAB 5 will require Java 21.
```

Comment:
Could be slightly rephrased to follow the same phasing as in line 44 (and dropping "upcoming"):
```suggestion
Java 21 is required for openHAB 5.
```